### PR TITLE
implements the brand & asset integration into the Core Add-On

### LINKED ISSUE #146: "leave study" consent needs copy

### LINKED ISSUE #196: incorporate SimplySecure Privacy Policy redesign into the Core Add-On
This is a stub for work that will need to be done to incorporate the outcome of the Privacy Policy design work.

Working doc: https://docs.google.com/document/d/1ndysAp0gtMoQRZ82-fuhhjGKzWIv5r-05aMSr-amtg8/edit?usp=sharing

### LINKED ISSUE #219: copy changes for the core add-on via Threespot

### LINKED ISSUE #231: Brand asset replacement in core add-on via ThreeSpot
This is a stub to be filled out later.

### LINKED ISSUE #238: update Privacy Notice to contain information about product data collection
This is a consequence of the participant journey analytics proposal (#131).

### PR DESCRIPTION
**Pull request by hamilton**
Closes #238, closes #231, closes #219, closes #196, closes #146

This PR obviously touches a lot of files, but most of them are either asset additions / deletions / swaps or CSS changes.

In future PRs, I'll tackle these:

1. study card redesign
2. arrow placement
3. adding an "update your profile" page
4. changing the icon(s) to the flag

cc @changecourse who has already helped me do a very thorough design review.

Checklist for reviewer:

- [ ] The description should reference a bug or github issue, if relevant.
- [ ] There must be a [`CHANGELOG.md`](./CHANGELOG.md) entry for any non-test change.
- [ ] Any change to the NPM commands must be carefully reviewed to make sure it won't break the Add-ons pipeline.
- [ ] Any version increase must follow the [release process](./docs/RELEASE_PROCESS.md).

### CODE REVIEW

File: src/routes/current-studies/StudyCard.svelte
Comment (hamilton) [inline src/routes/current-studies/StudyCard.svelte:141]: note: we will tackle the study consent needs in a follow-on PR.

**Comment by hamilton** [comment]
@rhelmer I'm noticing that the integration test seems to work on my machine but not in CI. Any clues about why that might be the case?

File: package.json
Comment (hamilton) [inline package.json:14]: this aligns the storybook pathing with the actual builid

File: tests/core-addon/integration/core_addon.js
Comment (hamilton) [inline tests/core-addon/integration/core_addon.js:81]: I believe this is the only functional change needed in this PR.

**Comment by hamilton** [comment]
@Dexterp37 mind doing a review of this PR? I've chatted with @rhelmer about the failing CI. The integration test seems to failing there even though it works locally on my machine.

**Comment by rhelmer** [comment]
> @rhelmer I'm noticing that the integration test seems to work on my machine but not in CI. Any clues about why that might be the case?

I started debugging this in https://github.com/mozilla-rally/rally-core-addon/pull/333 - I used @Dexterp37's screenshot example from https://github.com/mozilla-rally/rally-core-addon/issues/252 and the result is:

image

The failing test is checking that we can replace study contents using remote-settings and have the contents update immediately in the UI. The core add-on sends an `update-state` message along with the new state (`{enrolled, availableStudies}`). Is it possible this is broken in the Svelte app @hamilton ? If so we should add a unit test those are easier to debug :)

File: package.json
Comment (Dexterp37) [inline package.json:15]: Nit: commands here are not sorted, would you please move it back to where it was? :P

**Review by Dexterp37 — CHANGES_REQUESTED** [review]
This looks good overall. Please address the comment in `package.json` and rebase this. I'd love to take another look after this is rebased on the most recent master.

**Comment by Dexterp37** [comment]
> The failing test is checking that we can replace study contents using remote-settings and have the contents update immediately in the UI. The core add-on sends an `update-state` message along with the new state (`{enrolled, availableStudies}`). Is it possible this is broken in the Svelte app @hamilton ? If so we should add a unit test those are easier to debug :)

Thanks for taking the screenshot, this definitely put me on the right path. I believe that the update mechanism works, but the test is slightly flaky: I found that there's a possibility the test would fail on slow machines (e.g. CI). I have a potential fix in #334.

@hamilton what about we get #334 reviewed and merged first, then you can rebase on master and we can validate the above theory? Even though things work locally, I'd rather not merge something that breaks our CI.

**Comment by hamilton** [comment]
@Dexterp37 sounds good to me. Thank you & @rhelmer for investigating this.

**Review by Dexterp37 — APPROVED** [review]
r+ -  but let's not merge yet. Please, let's rebase on #334 once that's merged and make sure CI is green before merging. We can't afford to have broken integration tests with our schedule :)

**Pull request merged**
